src/vision/.pytest_cache/v/cache/lastfailed

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
wtf is this shit, Simon???

Reply:
srry, supposed only to request from the AI team, accidentally send to u